src/App.vue

```diff
@@ -153,6 +153,13 @@ export default {
     let config;
```

Comment:
Just reload the page on an hash change event.
```suggestion
    let config;
    window.onhashchange = function() { location.reload(); };
```